pages/exhibition/_.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div
+    data-qa="exhibition page"
+  >
+    <HeroImage
+      v-if="hero"
+      :image-url="heroImage.url"
+      :image-content-type="heroImage.contentType"
+      :header="header"
+      :lead="lead"
+      :identifier="hero.identifier"
+      :citation="hero.citation"
+      :rights-statement="hero.license"
+      :name="hero.name"
+      :provider="hero.provider"
+      :creator="hero.creator"
+      :url="hero.url"
+    />
+    <b-container class="pb-3">
+      <b-row>
+        <b-col
+          cols="9"
+          class="pb-3"
+        >
+          <article>
+            <!-- eslint-disable vue/no-v-html -->
+            <div
+              data-qa="exhibition text"
+              v-html="mainContent"
+            />
+            <!-- TODO: remove when credits go to their own page? -->
+            <div
+              v-html="credits"
+            />
+            <!-- eslint-enable vue/no-v-html -->
+          </article>
+        </b-col>
+      </b-row>
+      <b-row>
+        <b-col>
+          <!-- TODO: add links to chapters, remove h2 -->
+          <h2>Chapters</h2>
```

Comment:
Should be localised

Reply:
This is only a placeholder, it'll be replaced by an actual component when we implement EC-3886. I don't think it's worth adding to the string to the locale file if it'll be removed later.